python/tskit/trees.py

```diff
@@ -4295,6 +4295,34 @@ def nexus(self, precision=14):
         s += "END;\n"
         return s
 
+    def to_macs(self, path_or_buf=None):
```

Comment:
We're not testing the ``path_or_buf`` argument here, so I'd say let's not bother with it. Just return a string - we can add this type of argument later, if we need it (and should try to be consistent across the formats).